identity/handler.go

```diff
@@ -509,6 +510,41 @@ type AdminCreateIdentityImportCredentialsOidcProvider struct {
 	//
 	// required: false
 	UseAutoLink bool `json:"use_auto_link,omitempty"`
+
+	// The organization to assign for the provider.
+	Organization uuid.NullUUID `json:"organization,omitempty"`
+}
+
+// Create Identity and Import SAML Credentials
+//
+// swagger:model identityWithCredentialsSaml
+type AdminIdentityImportCredentialsSAML struct {
+	// Configuration options for the import.
+	Config AdminIdentityImportCredentialsSAMLConfig `json:"config"`
+}
+
+// swagger:model identityWithCredentialsOidcConfig
+type AdminIdentityImportCredentialsSAMLConfig struct {
+	// A list of SAML Providers
+	Providers []AdminCreateIdentityImportCredentialsSAMLProvider `json:"providers"`
+}
+
+// Create Identity and Import SAML Credentials Configuration
+//
+// swagger:model identityWithCredentialsSamlConfigProvider
+type AdminCreateIdentityImportCredentialsSAMLProvider struct {
+	// The subject (`sub`) of the OpenID Connect connection. Usually the `sub` field of the ID Token.
```

Comment:
Should this still reference OIDC?

Reply:
Good catch, I missed the code docs in copy paste.